src/UpdateCapableInterface.php

```diff
@@ -18,11 +18,12 @@
      *
      * @since [*next-version*]
      *
-     * @param array                           $changeSet An array of changes, mapping field name to changed value.
-     * @param LogicalExpressionInterface|null $condition An optional condition which, if specified, restricts the
-     *                                                   affected records to those that satisfy this condition.
+     * @param array|TermInterface[]|Traversable changeSet The change set, mapping field names to their new values or
```

Comment:
Here, on the other hand, `array` looks too wide. What does it accept? An array of containers, or an array of iterables?

Reply:
> The change set, mapping field names to their new values or value expressions.

Example:

```
[
	'field1' => 'new value',
	'field2' => $term
]
```

Reply:
That's an example with an array, but doesn't answer my question.

Reply:
The accepted type is an array or traversable of scalar values or `TermInterface` instances. The example was meant to show that there is no nesting.

Reply:
> The accepted type is an array or traversable

Reply:
Doesn't it? `array|TermInterface[]|Traversable`

And please don't just reply with "no". If "no", explain why "no".

Reply:
See f0dd261678046198cddcb9831b5c66bac2a2bef2.